packages/thirdweb/src/pay/buyWithCrypto/actions/getHistory.ts

```diff
@@ -0,0 +1,88 @@
+import type { ThirdwebClient } from "../../../client/client.js";
+import { getClientFetch } from "../../../utils/fetch.js";
+import { getPaySwapHistoryEndpoint } from "../utils/definitions.js";
+import type { BuyWithCryptoStatus } from "./getStatus.js";
+
+/**
+ * The parameters for [`getBuyWithCryptoHistory`](https://portal.thirdweb.com/references/typescript/v5/getBuyWithCryptoHistory) function
+ * It takes the wallet history address and optional cursor and page size. for paginated results.
+ */
+export type WalletSwapHistoryParams = {
```

Comment:
these param names should match the function names

```suggestion
export type GetBuyWithCryptoHistoryParams = {
```